cpu/esp32/Makefile.include

```diff
@@ -7,9 +7,7 @@ endif
 
 ESP_SDK_DIR = $(ESP32_SDK_DIR)
 
-# With the '-Os' option, the ESP32 hangs sporadically in 'tests/bench*' if
-# interrupts are disabled too early by benchmark tests.
-CFLAGS_OPT ?= -O2
+CFLAGS_OPT ?= -Os
```

Comment:
```suggestion
```
`CFLAGS_OPT` is set to `-Os` by default in https://github.com/RIOT-OS/RIOT/blob/8ffd34ddd05627e44193945e8365aeca07899875/cpu/esp_common/Makefile.include#L45